src/doc/src/reference/unstable.md

```diff
@@ -1233,63 +1233,147 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
 * RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
 
-The `workspace-inheritance` feature allows workspace members to inherit fields
-and dependencies from a workspace.
+### The `workspace.package` table
 
-Example 1:
+*Stabilization*: This would be in [`workspaces.md`][workspaces], under
+[The `workspace.metadata` table][workspace-metadata-table]
 
-```toml
-# in workspace's Cargo.toml
-[workspace.dependencies]
-log = "0.3.1"
-log2 = { version = "2.0.0", package = "log" }
-serde = { git = 'https://github.com/serde-rs/serde' }
-wasm-bindgen-cli = { path = "crates/cli" }
-```
+The `workspace.package` table is where you define keys that can be
+inherited by members of a workspace. These keys can be inherited by
+defining them in the member package with `{key}.workspace = true`.
 
-```toml
-# in a workspace member's Cargo.toml
-[dependencies]
-log.workspace = true
-log2.workspace = true
-```
+Keys that are supported:
 
-Example 2:
+| A - I           | J - Z          |
+|-----------------|----------------|
+| `authors`       | `keywords`     |
+| `categories`    | `license`      |
+| `description`   | `license-file` |
+| `documentation` | `publish`      |
+| `edition`       | `readme`       |
+| `exclude`       | `repository`   |
+| `homepage`      | `rust-version` |
+| `include`       | `version`      |
+
+For Example:
 ```toml
-# in workspace's Cargo.toml
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
+
 [workspace.package]
 version = "1.2.3"
 authors = ["Nice Folks"]
 description = "..."
 documentation = "https://example.github.io/example"
-readme = "README.md"
-homepage = "https://example.com"
-repository = "https://github.com/example/example"
-license = "MIT"
-license-file = "./LICENSE"
-keywords = ["cli"]
-categories = ["development-tools"]
-publish = false
-edition = "2018"
 ```
 
 ```toml
-# in a workspace member's Cargo.toml
+# [PROGECT_DIR]/bar/Cargo.toml
 [package]
+name = "bar"
 version.workspace = true
 authors.workspace = true
 description.workspace = true
 documentation.workspace = true
-readme.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
-license-file.workspace = true
-keywords.workspace = true
-categories.workspace = true
-publish.workspace = true
 ```
 
+
+### The `workspace.dependencies` table
+
+The `workspace.dependencies` table is where you define dependencies to be
+inherited by members of a workspace. Inheriting a dependency is as simple as
+[specifying the dependency][specifying-dependencies] in this table then
+[inheriting that dependency][inheriting-a-dependency-from-a-workspace] in the member
+`[dependencies]` table.
```

Comment:
This second sentence still needs some re-work.  "its as simple as" implies its the same but its not.  I'd at least say "similar to"  and make this the introduction to the caveats below